python/poverty-processing.py

```diff
@@ -0,0 +1,91 @@
+# Take user input for the state
+
+print "What state would you like to lean about?"
+print "Options: 1) Massachusetts, 2) Washington"
+
+acceptable = False
+while acceptable == False:
```

Comment:
Looks good - How would you feel about trying a more OO approach with some methods and things like that